app/controllers/prior_authority/applications_controller.rb

```diff
@@ -0,0 +1,33 @@
+module PriorAuthority
+  class ApplicationsController < ApplicationController
+    before_action :authenticate_provider!
+    layout 'prior_authority'
+
+    def index
+      @model = current_provider.prior_authority_applications.draft
+    end
+
+    def create
+      initialize_application do |paa|
+        redirect_to edit_prior_authority_steps_prison_law_path(paa)
+      end
+    end
```

Comment:
Cheers, I did think about this.

There is the prison law question which goes first and does need to be stored. So there is already going to be a need for the record and I personally don't see a problem with storing the value for simplicities sake (data is cheap) and then deleting it when they click finish or having a cleanup job of "pre_drafts" somewhere down the line. Alternatively, we may want to keep to keep it as an indicator of how many try to apply and answer "Yes", either incorrectly or because they thought they could apply for PA of less than that amount.